components/dashboard/src/data/workspaces/update-workspace-description-mutation.ts

```diff
@@ -27,19 +27,14 @@ export const useUpdateWorkspaceDescriptionMutation = () => {
             // pro-actively update workspace description rather than reload all workspaces
             queryClient.setQueryData<ListWorkspacesQueryResult>(queryKey, (oldWorkspacesData) => {
                 return oldWorkspacesData?.map((info) => {
-                    if (info.workspace.id !== workspaceId) {
+                    if (info.id !== workspaceId) {
                         return info;
                     }
 
                     // TODO: Once the update description response includes an updated record,
                     // we can return that instead of having to know what to merge manually (same for other mutations)
-                    return {
-                        ...info,
-                        workspace: {
-                            ...info.workspace,
-                            description: newDescription,
-                        },
-                    };
+                    info.name = newDescription;
```

Comment:
Is it ok, that we return the same object here? Not sure why did we retutn a new object before?

Reply:
Not sure why we do it before, changed code in case break something